### PR TITLE
Minor L2 networks doc update

### PR DESCRIPTION
**Pull request by andrijapanicsb**

### CODE REVIEW

**Comment by andrijapanicsb** [comment]
requesting docbuild

**Comment by andrijapanicsb** [comment]
@nvazquez @anuragaw @shwstppr - kindly LGTM - just image added.

File: source/adminguide/networking.rst [hunk 1]
```diff
@@ -105,6 +105,10 @@ IP addresses.
 -  Userdata and metadata can be passed to the VM using a config drive
    (which must be enabled in the network service offering)
 
+Example GUI dialog box (for a regular user account) is shown below:
+
+|L2-networks-gui.JPG|
```
Comment (shwstppr) [inline source/adminguide/networking.rst]: Screenshot with only the dialog will look nicer

**Comment by andrijapanicsb** [comment]
requesting docbuild

**Pull request merged**